python/ray/_private/usage/usage_lib.py

```diff
@@ -144,6 +160,23 @@ def _generate_cluster_metadata():
     return metadata
 
 
+def print_usage_stats_heads_up_message() -> None:
+    try:
+        if _usage_stats_prompt_disabled() or _usage_stats_enabled():
+            return
+
+        if _is_in_third_party_library():
+            return
+
+        print(
+            "Usage stats collection will be enabled by default in the next release. "
+            "See https://docs.google.com/document/d/1ZT-l9YbGHh-iWRUC91jS-ssQ5Qe2UQ43Lsoc1edCalc/edit#heading=h.17dss3b9evbj for more details"
+        )
```

Comment:
this should go to stderr

Reply:
Also can you link to the github issue?

Reply:
Any reason why stderr?

Reply:
What's your thoughts on having a ray.io/usage-stats page? https://github.com/ray-project/ray/pull/22985#discussion_r823343459